elfpy/utils/apeworx_integrations.py

```diff
@@ -209,6 +212,122 @@ def get_pool_state(tx_receipt: ReceiptAPI, hyperdrive_contract: ContractInstance
     return pool_state
 
 
+PoolInfo = namedtuple("PoolInfo", ["start_time", "block_time", "term_length", "market_state"])
+
+
+# TODO: remove commenets after verifying the accuracy of this function through more testing
+def get_agent_deltas(tx_receipt: ReceiptAPI, trade, addresses, trade_type, pool_info: PoolInfo):
+    """Get the change in an agent's wallet from a transaction receipt."""
+    agent = tx_receipt["operator"]  # type: ignore
```

Comment:
This apeworx_integrations file might just need to include `# pyright: reportOptionalMemberAccess=false, reportGeneralTypeIssues=false` at the top so that we don't have to put `# type: ignore` everywhere.

Reply:
object.attribute works here